app/src/test/java/com/github/se/signify/model/challenge/ChallengeTest.kt

```diff
@@ -16,38 +16,45 @@ class ChallengeTest {
     assertEquals("", challenge.mode)
     assertEquals("pending", challenge.status)
     assertEquals(1, challenge.round)
-    assertEquals(0, challenge.player1Score)
-    assertEquals(0, challenge.player2Score)
-    assertEquals("", challenge.currentGesture)
-    assertEquals(emptyMap<String, String>(), challenge.responses)
+    assertEquals(emptyList<Long>(), challenge.player1Times)
+    assertEquals(emptyList<Long>(), challenge.player2Times)
+    assertEquals(listOf(false, false, false), challenge.player1RoundCompleted)
+    assertEquals(listOf(false, false, false), challenge.player2RoundCompleted)
+    assertEquals("not_started", challenge.gameStatus)
   }
 
   @Test
```

Comment:
There are some edge cases that are not taken into account in these tests : such as when both players complete a challenge simultaneously or a player leaves the challenge mid-way.

Reply:
As I already have a 88% line coverage, I think i will focus on this for a next task.